specs/waku/waku-v2.md

```diff
@@ -190,11 +190,19 @@ message RPC {
 }
 
 message HistoryQuery {
-  // TODO Include time range, topic/contentTopic, limit, cursor, (request-id), etc
+  // Represents the current request UUID, used to identify the corresponding response.
```

Comment:
Why not be consistent with current spec format? All fields are explained in section below.

Reply:
There's even a TODO under the sections for these messages